UnityProject/Assets/Scripts/Objects/Botany/SeedExtractor.cs

```diff
@@ -156,10 +159,12 @@ public bool WillInteract(HandApply interaction, NetworkSide side)
 			   interaction.HandObject.TryGetComponent<GrownFood>(out _);
 		}
 
+		#region IAPCPowerable
+
 		/// <summary>
-		/// IS NOT USED BUT REQUIRED BY IAPCPowered
+		/// IS NOT USED BUT REQUIRED BY IAPCPowerable
```

Comment:
I hate this

Reply:
The comment in general, or the new name?

Reply:
the need to implement a method from an interface that is not used. That interface shouldn't exists, power state changes could be handled with events... Sorry, just ranting.